Nodes/common.mk

```diff
@@ -3,4 +3,36 @@ ROOT_DIR = $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 EXTERNAL_MODULE_DIRS := $(ROOT_DIR)/modules
 RIOTBASE ?= $(ROOT_DIR)/RIOT
 
+JSON_OUTPUT = node.json
+UUID_OUTPUT = uuid.txt
+QRCODE = qrcode.png
+
+# hardcoded wow, many secure, much professional, such cool
+PSK_ID = hardcoded_id
+PSK = hardcoded_psk
```

Comment:
Why not generate them randomly? 
https://linuxhint.com/generate-random-string-bash/

Seeing that link, the kernel provides uuid, maybe useful for the UUID without python ;)